lib/rspec/rails/example/mailer_example_group.rb

```diff
@@ -22,7 +22,7 @@ module MailerExampleGroup
         included do
           include ::Rails.application.routes.url_helpers
           options = ::Rails.configuration.action_mailer.default_url_options
-          options.each { |key, value| default_url_options[key] = value } if options
+          options&.each { |key, value| default_url_options[key] = value }
```

Comment:
What versions of Ruby does that operator support?

Reply:
[>= 2.3](https://www.ruby-lang.org/en/news/2015/12/25/ruby-2-3-0-released/).. v4 will drop support for ruby below 2.3, so using lonely operator should be safe..